Bio/NeuralNetwork/Gene/Schema.py

```diff
@@ -470,7 +470,7 @@ def __init__(self, num_schemas=100,
 
         Arguments:
 
-        o num_schemas -- the number of useful (positive fitness) schemas
+        * num_schemas -- the number of useful (positive fitness) schemas
```

Comment:
Using ``*`` is valid RST, but for consistency with previous examples elsewhere in Biopython please use a minus sign.

Also, you must intent the next line by another two spaces as right now this is not valid RST.